src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age
+      ? <p className="Person__age">{`I am ${person.age}`}</p>
+      : null
+    }
+    <p className="Person__partner">
+      {(() => {
+        if (person.isMarried) {
+          return person.sex === 'm'
+            ? `${person.partnerName} is my wife`
+            : `${person.partnerName} is my husband`;
+        }
+
+        return `I am not married`;
```

Comment:
you shouldn't use a function here. just use conditional rendering with logical `&&`
https://reactjs.org/docs/conditional-rendering.html

Reply:
and use destructuring (checklist)
<img width="628" alt="image" src="https://user-images.githubusercontent.com/102411130/204741258-27941949-eb15-4739-8dd3-ed1c15599545.png">
<img width="461" alt="image" src="https://user-images.githubusercontent.com/102411130/204741296-e7458279-a08e-416b-a76d-f09afbb2e3f5.png">